src/DependencyInjection/Extension.php

```diff
@@ -125,8 +125,9 @@ private function prepareDoctrineBundle(array $config, LoaderInterface $loader, C
             }
         }
 
+        $userMappingKey =  $classMapping[User::class] ?? User::class;
         $entityEmailFieldMapping = $primaryEntityEmailFieldMapping = [
-            $classMapping[User::class] => 'email',
```

Comment:
```php
$entityEmailFieldMapping = $primaryEntityEmailFieldMapping = [];
if (isset($classMapping[User::class])) {
    $entityEmailFieldMapping[User::class] = 'email';
    $primaryEntityEmailFieldMapping[User::class] = 'email';
}
```

Reply:
we then also need to verify what happens in `SqlEmailLookup` if no entities are given (should quick return false i guess), given we want to keep the service unconditionally.